### DISCUSSION TITLE
Intel NUC8i5 - hw Transcoding issues.

### DISCUSSION THREAD

**Discussion by mintyman** [tteck/Proxmox#916] [Help Needed]
Should the Plex LXC script work out of the box with Intel NUC / 8th Gen igpu? (Hardware Transcoding )
I have spun it up to give it a try and haven't had any luck.

Only success so far has been running Proxmox >> Ubuntu LXC >> Docker >> Plex 

or Proxmox >> Ubuntu VM >> Plex.

I have tried to follow a few guides so far but really like the idea of using these Scripts to automate the process of spinning up LXC containers.

**Comment by tteck** [comment]
Maybe a driver? Read through this post https://github.com/tteck/Proxmox/issues/8
